docs/dev-machine-setup.md

```diff
@@ -21,7 +21,28 @@ outlined in the [Getting Started](getting-started.md) instructions).
 
 ### AWS
 
-`[TODO]`
+Firecracker development environment on AWS can be setup using bare metal instances. Follow these steps to create a bare metal instance.
+
+1. If you don't already have an AWS account, create one: https://portal.aws.amazon.com/billing/signup.
+1. Install AWS CLI: https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-install.html
```

Comment:
This step is not mandatory because you can start the instance from the AWS console. Also, it seems like you are not using the AWS cli anyway.